docs/source/onnx.rst

```diff
@@ -516,212 +242,205 @@ And below is the list of unsupported patterns for LHS indexing. ::
-  dynamic lookups are not available.
-
-* PyTorch and ONNX backends(Caffe2, ONNX Runtime, etc) often have implementations of operators with some
-  numeric differences.  Depending on model structure, these differences
-  may be negligible, but they can also cause major divergences in behavior
-  (especially on untrained models.)  We allow Caffe2 to call directly to Torch implementations of operators, to
-  help you smooth over these differences when precision is important,
-  and to also document these differences.
-
 Supported operators
 -------------------
 
 The following operators are supported:
 
-* BatchNorm
-* ConstantPadNd
-* Conv
-* Dropout
-* Embedding (no optional arguments supported)
-* EmbeddingBag
-* FeatureDropout (training mode not supported)
-* Index
-* MaxPool1d
-* MaxPool2d
-* MaxPool3d
-* RNN
-* abs
-* absolute
-* acos
-* adaptive_avg_pool1d
-* adaptive_avg_pool2d
-* adaptive_avg_pool3d
-* adaptive_max_pool1d
-* adaptive_max_pool2d
-* adaptive_max_pool3d
-* add (nonzero alpha not supported)
-* addmm
-* and
-* arange
-* argmax
-* argmin
-* asin
-* atan
-* avg_pool1d
-* avg_pool2d
-* avg_pool2d
-* avg_pool3d
-* as_strided
-* baddbmm
-* bitshift
-* cat
-* ceil
-* celu
-* clamp
-* clamp_max
-* clamp_min
-* concat
-* copy
-* cos
-* cumsum
-* det
-* dim_arange
-* div
-* dropout
-* einsum
-* elu
-* empty
-* empty_like
-* eq
-* erf
-* exp
-* expand
-* expand_as
-* eye
-* flatten
-* floor
-* floor_divide
-* frobenius_norm
-* full
-* full_like
-* gather
-* ge
-* gelu
-* glu
-* group_norm
-* gt
-* hardsigmoid
-* hardswish
-* hardtanh
-* im2col
-* index_copy
-* index_fill
-* index_put
-* index_select
-* instance_norm
-* interpolate
-* isnan
-* KLDivLoss
-* layer_norm
-* le
-* leaky_relu
-* len
-* log
-* log1p
-* log2
-* log_sigmoid
-* log_softmax
-* logdet
-* logsumexp
-* lt
-* masked_fill
-* masked_scatter
-* masked_select
-* max
-* mean
-* min
-* mm
-* mul
-* multinomial
-* narrow
-* ne
-* neg
-* new_empty
-* new_full
-* new_zeros
-* nll_loss
-* nonzero
-* norm
-* ones
-* ones_like
-* or
-* permute
-* pixel_shuffle
-* pow
-* prelu (single weight shared among input channels not supported)
-* prod
-* rand
-* randn
-* randn_like
-* reciprocal
-* reflection_pad
-* relu
-* repeat
-* replication_pad
-* reshape
-* reshape_as
-* round
-* rrelu
-* rsqrt
-* rsub
-* scalar_tensor
-* scatter
-* scatter_add
-* select
-* selu
-* sigmoid
-* sign
-* sin
-* size
-* slice
-* softmax
-* softplus
-* sort
-* split
-* sqrt
-* squeeze
-* stack
-* std
-* sub (nonzero alpha not supported)
-* sum
-* t
-* tan
-* tanh
-* threshold (non-zero threshold/non-zero value not supported)
-* to
-* topk
-* transpose
-* true_divide
-* type_as
-* unbind
-* unfold (experimental support with ATen-Caffe2 integration)
-* unique
-* unsqueeze
-* upsample_nearest1d
-* upsample_nearest2d
-* upsample_nearest3d
-* view
-* weight_norm
-* where
-* zeros
-* zeros_like
+Python built-in operators on torch.Tensors:
```

Comment:
Do we need this long list? The goal is to export the user's model, and if there is some operator that is not supported, we throw the errors in symbolic function (this is currently we do), and we can think to support it on judgement.